Modules/Loadable/Colors/Resources/UI/qSlicerColorsModuleWidget.ui

```diff
@@ -44,6 +44,18 @@
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="qMRMLColorTableComboBox" name="ColorTableComboBox">
+          <property name="enabled">
```

Comment:
@Punzo Have you made changes to qSlicerColorsModuleWidget.ui intentionally?

Reply:
No sorry didn't see.

Reply:
ok perfect you have already reverted them. Thanks!